src/leaflet.geometryutil.js

```diff
@@ -540,7 +540,7 @@ L.GeometryUtil = L.extend(L.GeometryUtil || {}, {
             var l1 = latlngs[i],
                 l2 = latlngs[i+1];
             portion = lengths[i];
-            if (L.GeometryUtil.belongsSegment(point, l1, l2)) {
+            if (L.GeometryUtil.belongsSegment(point, l1, l2, 0.0001)) {
```

Comment:
This value seems pretty arbitrary :) Where does it come from?

Reply:
It's arbitrary a little bit like 0.2 :) i can't put 0 because it's not possible to find the point on the line with this value, it should be because the point is on the line but i think that it's not perfectly on it.

Reply:
Gotcha!